packages/location-field/src/styled.js

```diff
@@ -138,9 +139,22 @@ export class MenuItem extends Component {
   };
 
   render() {
-    const { active, centeredText, children, disabled, header } = this.props;
+    const {
+      active,
+      centeredText,
+      children,
+      color,
+      disabled,
+      header
+    } = this.props;
     return header ? (
-      <MenuItemHeader centeredText={centeredText}>{children}</MenuItemHeader>
+      <MenuItemHeader
+        className="header"
```

Comment:
Is this classname set in CSS anywhere (including OTP-RR)? If not, you could convert the header into a styled component and specify the hover effect in there.

Reply:
Yes from what I can tell there are rules relating to `.header` menu times in OTP-RR